panel/io/compile.py

```diff
@@ -71,6 +73,65 @@ def check_cli_tool(tool_name):
         return False
 
 
+def find_module_bundles(module_spec: str) -> dict[pathlib.PurePath, list[ReactiveESM]]:
+    """
+    Takes module specifications and extracts a set of components to bundle.
+
+    Arguments
+    ---------
+    module_spec: str
+         Module specification either as a dotted module or a path to a module.
+
+    Returns
+    -------
+    Dictionary containing the bundle paths and list of components to bundle.
+    """
+    if re.match(r'^[A-Za-z]:\\', module_spec):
```

Comment:
```suggestion
    if module_spec[1:3] == r":\\":
```

Can't something like this not be used? Please double-check.